packages/schema/src/config/build.ts

```diff
@@ -38,6 +39,23 @@ export default defineUntypedSchema({
     },
   },
 
+  /**
+   * Log level when building logs.
+   *
+   * Defaults to 'quiet' when running in CI or when a TTY is not available.
+   * This option is then used as 'silent' in Vite and 'none' in Webpack
+   *
+   * @type {'quiet' | typeof import('../src/types/global/vite').ViteLogLevel | typeof import('../src/types/global/webpack').WebpackStats}
+   */
+  logLevel: {
+    $resolve: (val) => {
+      if (isMinimal) {
+        return 'quiet'
+      }
+      return val
```

Comment:
Probably we always want to respect an explicit value

```suggestion
      return val ?? (isMinimal ? 'quiet' : undefined)
```